app/models/user.rb

```diff
@@ -84,7 +84,7 @@ def self.find_or_create_from_cis2_oidc(userinfo)
     user.tap(&:save!)
   end
 
-  def is_medical_secretary?
+  def is_admin_staff?
```

Comment:
```suggestion
  def is_admin?
```

Could we also go with this? For consistency with `is_nurse?`.

Reply:
We could. It's nice and matches the length of "nurse", but also I was worried with this being mistaken with a "sysadmin", which is the other commonly used meaning. I was  hoping `admin_staff` would void confusion.

Do you think `admin` wouldn't be misleading?

Reply:
@tvararu How do you feel about calling this `admin` vs `admin_staff`? ☝️

Reply:
I prefer `admin`

Reply:
👍 Let's do it